logstash-core/src/test/java/org/logstash/secret/store/SecretStoreUtilTest.java

```diff
@@ -14,8 +14,9 @@
 public class SecretStoreUtilTest {
 
     private String asciiString;
+
     @Before
-    public void setup(){
+    public void _setup() {
```

Comment:
why prefix with _ ?

Reply:
This is strictly for my IDE... i use an auto-sort (notice all the method names are alpha sorted), and like to keep the `setup` method on the top. I try to avoid this type non-sense except for test code.